### ISSUE TITLE
Rename to mamba-github-action

### ISSUE THREAD

**Issue by SylvainCorlay** [TheSnakePit/setup-mamba#8]
The name of the repository does not make it clear what it is about IMO.

**Comment by jtpio** [comment]
`setup-*` seems to be a common naming convention for many third-party GitHub actions.
Usually it is enough to know what it refers to for those already using GitHub actions.

However the repository should have a project description to mention what it is.

**Comment by hadim** [comment]
Pinging here to see if this GH action is still being developed or not? It looks like using [micromamba ](https://github.com/mamba-org/mamba/pull/506) and mamba could dramatically speed up a lot of CI jobs.

**Comment by jtpio** [comment]
@hadim yes it is still being developed but there is no release yet (it was temporarily paused due to Windows support: https://github.com/mamba-org/setup-mamba/issues/4)

Sounds like we should indeed switch to micromamba!

**Comment by hadim** [comment]
Would that be worth combining the effort with https://github.com/conda-incubator/setup-miniconda? See also https://github.com/conda-incubator/setup-miniconda/issues/75

**Comment by Borda** [comment]
hi, I d love to try mamba or our CI, so what GH action is "official"?
